marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/provenance/pruning.sjs

```diff
@@ -26,7 +26,7 @@ function validatePruneRequest({ retainDuration, batchSize = 100 }) {
 function pruneProvenance(pruneRequest, endpointState) {
     xdmp.securityAssert("http://marklogic.com/data-hub/privileges/prune-provenance", "execute");
```

Comment:
We should also change this privilege to be delete-provenance, along with this test file name and function names.